TypeCobol.Test/Parser/Programs/Cobol2002/Typedef5.Mix.txt

```diff
@@ -6,6 +6,7 @@
        01 A2   TYPEDEF STRICT PUBLIC comp-2.
        01 A3   TYPEDEF STRICT PUBLIC pointer.
        01 A4   TYPEDEF STRICT PUBLIC procedure-pointer.
+Line 9[11,12] <27, Error, Syntax> - Syntax error : A group item cannot be empty. Add children, picture or usage declaration.
        01 A5   TYPEDEF STRICT PUBLIC comp-5.
```

Comment:
As discussed I think we should have the same test on allowed usages on both data declarations and typedefs because allowing suche typedefs (with invalid usages) would lead to non-compiling generated cobol.

Reply:
We can remove the test in `Cobol2002Checker.cs`.

Reply:
Test removed